web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/CertificateEventMessageValidatorImpl.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygstjanst.web.service.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygstjanst.web.service.CertificateEventMessageValidator;
+
+@Service
+@Slf4j
+public class CertificateEventMessageValidatorImpl implements CertificateEventMessageValidator {
+
+    private static final String MESSAGE_SENT = "message-sent";
+
+    public boolean validate(String eventType, String certificateId, String messageId) {
```

Comment:
Maybe this implementation can be simplified. Now it validates and throws an exception that it catches itself.

Instead of throwing and catching. Maybe just log the error and return false in each if-statement and return true at the end of the method?